src/Workleap.OpenApi.MSBuild/SwaggerManager.cs

```diff
@@ -45,11 +45,14 @@ public async Task InstallSwaggerCliAsync(CancellationToken cancellationToken)
         {
             return;
         }
-        
+
         var retryCount = 0;
         while (retryCount < 2)
         {
-            var result = await this._processWrapper.RunProcessAsync("dotnet", new[] { "tool", "update", "Swashbuckle.AspNetCore.Cli", "--ignore-failed-sources", "--tool-path", this._swaggerDirectory, "--configfile", Path.Combine(this._openApiToolsDirectoryPath, "nuget.config"), "--version", SwaggerVersion }, cancellationToken);
+            var result = await this._processWrapper.RunProcessAsync(
+                "dotnet",
+                new[] { "tool", "update", "Swashbuckle.AspNetCore.Cli", "--ignore-failed-sources", "--tool-path", this._swaggerDirectory, "--configfile", Path.Combine(this._openApiToolsDirectoryPath, "nuget.config"), "--version", SwaggerVersion },
```

Comment:
nit: You can be fancy here too :)

```suggestion
                ["tool", "update", "Swashbuckle.AspNetCore.Cli", "--ignore-failed-sources", "--tool-path", this._swaggerDirectory, "--configfile", Path.Combine(this._openApiToolsDirectoryPath, "nuget.config"), "--version", SwaggerVersion],
```

Reply:
Indeed, I must admit I commited your proposed changes through Github but didn't go through all the code 😅